library/src/reduction/reduction_solution_impl.hpp

```diff
@@ -214,24 +221,58 @@ namespace hiptensor
                                                                           PropagateNan,
                                                                           OutputIndex>;
 
-        std::vector<DeviceOpPtr> opPtrs;
-        ck::tensor_operation::device::instance::add_device_reduce_instance_blockwise<
-            InDataType,
```

Comment:
`add_device_reduce_instance_blockwise` creates some instances that generate wrong output. 

Now we only use two instances that we trust.